### DISCUSSION TITLE
Tray icon improvements

### DISCUSSION THREAD

**Discussion by naixx** [seerge/g-helper#2657] [Ideas]
I've created a repo with tray icon improved. It show current temperature and MHz. Are you interested in a PR?
![image](https://github.com/seerge/g-helper/assets/1759282/8ec1558a-8381-48be-b9f8-5ffdbf11f9dd)

**Comment by seerge** [comment]
@naixx hello, I was planning to add some simple overlay showing basic readings somewhere in future, so I don't think that tray icon output is necessary on this stage. Thanks.
